css_tasks/task10/js/app.js

```diff
@@ -0,0 +1,18 @@
+import {
+  submitForm,
+  removeDisabledState,
+  addDisabledState,
+  hideFormText
+} from "./form.js";
+import { constants } from "./constants.js";
+const { signupForm, formButton, inputs, formText } = constants;
+
+// EVENTS
+signupForm.addEventListener("submit", submitForm);
+inputs.forEach(item =>
+  item.addEventListener("focus", () =>
+    removeDisabledState(formButton, formText)
+  )
+);
+signupForm.addEventListener("submit", () => addDisabledState(formButton));
+formButton.addEventListener("click", () => hideFormText(formText));
```

Comment:
here you create 3 listeners in place where it possible to create 1

what if user press enter to submit form and not click on button?

Reply:
Actually it worked the same, but I get your point

Reply:
Changed